### ISSUE TITLE
How to implement this in R?

### ISSUE THREAD

**Issue by Kabouik** [jawj/OverlappingMarkerSpiderfier-Leaflet#36]
Hi,

I am using Leaflet through _R_ and would need OverlappingMarkerSpiderfier-Leaflet for the map I want to plot, but I don't think this is implemented in the Leaflet _R_ package. Is it planned? In the meantime, is there a way to implement it manually without knowing anything about JS? Or can I edit one of the output files I get from _R_ when rendering my project as a web page?

**Comment by nicocriscuolo** [comment]
Yes, it would be really cool to implement this function also in R.

Thank you in advance!
